src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,63 @@
+.icon {
+  display: block;
+  width: $icon-width;
+  height: $icon-width;
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  @include onTablet {
+    width: $icon-width / 3 * 4;
+    height: $icon-width / 3 * 4;
+  }
+
+  &__tooltip {
+    display: block;
+    margin-right: 30px;
+    padding-right: 12px;
+
+    position: relative;
+    top: 50%;
+    transform: translate(-100%, -50%);
+    width: max-content;
+
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    opacity: 0;
+    pointer-events: none;
+    transition: opacity 0.2s linear;
```

Comment:
I guess it is better to use same transition-duration value for all transitions on one web-site

Reply:
Yeas, I just forgot about this row )